ecommerce/src/main/kotlin/io/hhplus/ecommerce/checkout/application/consumer/CheckoutQueueConsumer.kt

```
@@ -0,0 +1,119 @@
+                userId = payload.userId,
+                eventType = SseEventType.CHECKOUT_COMPLETED,
+                data = result
+            )
+
+            val processingTime = System.currentTimeMillis() - startTime
+            logger.info {
+                "[CheckoutQueueConsumer] 처리 완료: requestId=${payload.requestId}, " +
+                    "orderId=${checkoutSession.orderId}, processingTime=${processingTime}ms"
+            }
+
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)